contrib/subrepo-cheri-compiler-rt/.gitrepo

```diff
@@ -6,7 +6,7 @@
 [subrepo]
 	remote = https://github.com/CTSRD-CHERI/compiler-rt.git
 	branch = monorepo-mirror
-	commit = 85b1bb71336940c89a0eda5a15b297a7440e4f77
+	commit = c34d2cb2e4a9fee4b2d668a6218504896269f8db
 	parent = d1984cf461afdc0661133909513dc255821d9c66
```

Comment:
Why was this not updated? Shouldn't it be a more recent commit? Otherwise git subrepo will got though that entire history again when updating next.

Reply:
Because c34d2cb2e4a9 is a downstream diff that still applies

Reply:
I could pull that into LLVM so git subrepo doesn't see a diff any more if you'd rather?

Reply:
No this doesn't matter - I was just surprised it hadn't changed.